lib/portus/errors.rb

```diff
@@ -37,5 +37,8 @@ class NotFoundError < RuntimeError; end
 
     # Raised if this client does not have the credentials to perform an API call.
     class CredentialsMissingError < RuntimeError; end
+
+    # Raised if any timeout reached
+    class TimeoutReachedError < RuntimeError; end
```

Comment:
Be careful, these errors are for registry-related exceptions, and this one right here is for the DB. I would move it to Portus::DB.

Reply:
Ok, I'll move it then. No worries. :)

Reply:
Done